src/pfe/portal/modules/Templates.js

```diff
@@ -54,7 +54,32 @@ module.exports = class Templates {
     }
 
     let newProjectTemplates = [];
-    await Promise.all(this.repositoryList.map(async function getTemplates(repository) {
+
```

Comment:
nit: I'd prefer this code in a subfunction, as this function is getting large

Reply:
moved